src/extension.ts

```diff
@@ -1812,8 +1812,13 @@ async function deleteContextKubernetes(explorerNode: explorer.KubernetesObject)
     }
 }
 
-function copyKubernetes(explorerNode: explorer.KubernetesObject) {
-    clipboard.writeSync(explorerNode.id);
+async function copyKubernetes(explorerNode: explorer.KubernetesObject) {
+    const clipboard = (<any>vscode.env).clipboard;
+    if (clipboard) {
```

Comment:
Again I would suggest a separate module exporting a 'copy text to clipboard' function that hides these details away.

Reply:
done.